test/unittests/analysis_test.cpp

```diff
@@ -37,15 +37,15 @@ TEST(analysis, example1)
 
 TEST(analysis, stack_up_and_down)
 {
-    const auto code = OP_DUP2 + 6 * OP_DUP1 + 10 * OP_POP + push(0);
+    const auto code = evmone::OP_DUP2 + 6 * evmone::OP_DUP1 + 10 * evmone::OP_POP + push(0);
     const auto analysis = analyze(rev, code);
 
     ASSERT_EQ(analysis.instrs.size(), 20);
     EXPECT_EQ(analysis.instrs[0].fn, op_tbl[OPX_BEGINBLOCK].fn);
-    EXPECT_EQ(analysis.instrs[1].fn, op_tbl[OP_DUP2].fn);
-    EXPECT_EQ(analysis.instrs[2].fn, op_tbl[OP_DUP1].fn);
-    EXPECT_EQ(analysis.instrs[8].fn, op_tbl[OP_POP].fn);
-    EXPECT_EQ(analysis.instrs[18].fn, op_tbl[OP_PUSH1].fn);
+    EXPECT_EQ(analysis.instrs[1].fn, op_tbl[evmone::OP_DUP2].fn);
```

Comment:
Need to remove all these changes, needed locally in order to compile and managed to commit in.